.goreleaser.yml

```diff
@@ -54,7 +54,7 @@ brews:
     homepage: "https://github.com/tektoncd/cli"
     description: Tekton CLI - The command line interface for interacting with Tekton
     test: |
-      system "#{bin}/tkn", "--version"
+      system "#{bin}/tkn", "version"
```

Comment:
oops that how it supposed to be. why it was not failing then :thinking: during release

Reply:
i don't think brew test was launched,